person.rb

```diff
@@ -1,14 +1,15 @@
 require './decorator'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
Getters and setters for rentals need to be set in order to read and change the values of the @rentals, so kindly add it to the attr_accessor instead of the attr_reader :+1: